dd-sdk-android/src/test/kotlin/com/datadog/android/log/LoggerTest.kt

```diff
@@ -52,16 +52,16 @@ internal class LoggerTest {
 
     @BeforeEach
     fun `set up`(forge: Forge) {
-        LogsFeature.envName = ""
-        LogsFeature.appVersion = ""
+        CoreFeature.envName = ""
+        CoreFeature.packageVersion = ""
```

Comment:
Is it still necessary to set those here? It seems it doesn't have any impact on the tests.